causalpy/pymc_models.py

```diff
@@ -85,10 +200,42 @@ def score(self, X, y) -> pd.Series:
 
 
 class WeightedSumFitter(ModelBuilder):
-    """Used for synthetic control experiments"""
+    """
+    Used for synthetic control experiments
+
+    .. note: Generally, the `.fit()` method should be rather than calling
+    `.build_model()` directly.
+
+    Defines the PyMC model:
+
+    - y ~ Normal(mu, sigma)
+    - sigma ~ HalfNormal(1)
+    - mu = X * beta
+    - beta ~ Dirichlet(1,...,1)
```

Comment:
It would be even better to get this as LaTeX formatted eqnarray. I believe we can do multi line equations with the math directive, https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#math

My preference would be to reorder the equations so the priors come first and the likelihood term is on the bottom line.